bgpd/bgp_routemap.c

```diff
@@ -4576,13 +4576,11 @@ DEFUN_YANG (no_match_probability,
 
 DEFPY_YANG (match_ip_route_source,
        match_ip_route_source_cmd,
-       "match ip route-source <(1-199)|(1300-2699)|WORD>",
+       "match ip route-source WORD",
        MATCH_STR
        IP_STR
        "Match advertising source address of route\n"
-       "IP access-list number\n"
-       "IP access-list number (expanded range)\n"
-       "IP standard access-list name\n")
+       "IP access-list name\n")
```

Comment:
cosmetic nit: Others have `IP Access-list name\n` (access-list => Access-list). Maybe let's do this for consistency?